canonical/ubuntu.com
# Refresh AT&T onboarding page

## Done

- Refresh AT&T onboarding page as per [copydoc](https://docs.google.com/document/d/1QG0zxy9u_oGhXJUBsm3mNQy0l5dLwWOlDrheB_-BPH8/edit)

## QA

- Visit https://ubuntu-com-12052.demos.haus/engage/unlisted/at&t-onboarding
- See that the page matches the copy doc, and looks good

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5997
